policy/modules/services/iiosensorproxy.te

```diff
@@ -0,0 +1,73 @@
+policy_module(iiosensorproxy)
+
+########################################
+#
+# iio-sensor-proxy (Debian package iio-sensor-proxy)
+# IIO sensors to D-Bus proxy
+# Industrial I/O subsystem is intended to provide support for devices
+# that in some sense are analog to digital or digital to analog convertors
+# .
+# Devices that fall into this category are:
+#  * ADCs
+#  * Accelerometers
+#  * Gyros
+#  * IMUs
+#  * Capacitance to Digital Converters (CDCs)
+#  * Pressure Sensors
+#  * Color, Light and Proximity Sensors
+#  * Temperature Sensors
+#  * Magnetometers
+#  * DACs
+#  * DDS (Direct Digital Synthesis)
+#  * PLLs (Phase Locked Loops)
+#  * Variable/Programmable Gain Amplifiers (VGA, PGA)
+
+########################################
+#
+# Declarations
+#
+
+type iiosensorproxy_t;
+type iiosensorproxy_exec_t;
+init_daemon_domain(iiosensorproxy_t, iiosensorproxy_exec_t)
+
+
+########################################
+#
+# Local policy
+#
+
+allow iiosensorproxy_t self:netlink_kobject_uevent_socket { bind create getattr setopt read };
+allow iiosensorproxy_t self:process { getsched setsched };
+allow iiosensorproxy_t self:unix_dgram_socket { create write };
+
+kernel_read_system_state(iiosensorproxy_t)
+
+dev_read_iio(iiosensorproxy_t)
+
+# for /sys/bus/iio/devices
+dev_read_sysfs(iiosensorproxy_t)
+
+# for writing to current_trigger
+dev_write_sysfs(iiosensorproxy_t)
```

Comment:
Can you be a little more explicit on what is being accessed? In the future I'd like to try to add more specific labeling in `/sys`, and this info would be useful.

Otherwise looks good to me.

Reply:
Is this good enough?

Reply:
Yes, thanks.